src/applications/accredited-representative-portal/tests/unit/utilities/api.unit.spec.jsx

```diff
@@ -66,13 +66,11 @@ describe('API utilities', () => {
       });
       fetchStub.resolves(successResponse);
 
-      await api.getPOARequests({ status: 'pending', sort: 'created_at_desc' });
+      await api.getPOARequests({ status: 'pending' });
```

Comment:
Revert the change to this line so that the test automatically complains again to us when fixed app code goes in.